tests/run-tests-on-external-cluster.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+: {$KUBEVIRTNAMESPACE:="openshift-cnv"}
+_kubectl=$(which kubectl)
+_oc=$(which oc)
+
+[[ -z ${_kubectl} && -z ${_oc} ]] && echo "OC or KUBECTL is not installed" && exit 1
+
+[ -z "${KUBECONFIG}" ] && echo "KUBECONFIG is not set" && exit 1
+
+cat << EOF > test_config.json
+{
+ "storageClassLocal": "hostpath-provisioner",
+ "storageClassHostPath": "hostpath-provisioner",
+ "storageClassRhel": "hostpath-provisioner",
+ "storageClassWindows": "hostpath-provisioner",
+ "manageStorageClasses": false
+}
+EOF
```

Comment:
@EdDev This is redundant. We don't have to use such configuration because the project already use default config file  tests/default-config.json.